ouroboros-network/test/Test/PeerState.hs

```diff
@@ -30,6 +30,7 @@ import           Control.Monad.Class.MonadThrow
 import           Control.Tracer
 
 import           Data.Semigroup.Action
+import           Ouroboros.Network.ErrorPolicy
```

Comment:
This is a rather strange commit. It causes the test suite to fail to compile because `Ouroboros.Network.ErrorPolicy`.